src/test/blk_create/README

```diff
@@ -0,0 +1,30 @@
+Linux NVM Library
+
+This is src/test/blk_create/README.
+
+This directory contains a unit test for pmemblk_create().
+
+The program in blk_create.c takes: path, bsize, poolsize, mode
+
+For example:
+
+	./blk_create /my/file 4096 20 0640
+
+This calls pmemblk_create() with the given arguments.  Note that poolsize
+is given in megabytes.
+
+Test cases:
+
+- expected to be successful
+	TEST0 non-existing file, poolsize > 0
+	TEST1 existing file, file length >= min required size, poolsize == 0
+
+- expected to fail
+	TEST2 non-existing file, poolsize==0
+	TEST3 existing file, file length >= min required size, poosize > 0
+	TEST4 existing file, file length < min required size, poosize == 0
+	TEST[56] two success cases above, but path is invalid (directory not exist)
```

Comment:
If we're talking about non-zero filled pools, there are already tests like that - maybe they could be a good starting point:
https://github.com/pmem/nvml/tree/master/src/test/blk_non_zero